server/app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,6 @@
+# Respond to API requests with a json response
+module Response
+  def json_response(object, status = :ok)
+    render json: object, status: status
```

Comment:
minor:
This is nice for getting familiar with concerns.

However, this doesn't really have any benefits besides some semantics:
`json_response({ message: e.message }, :not_found)`
vs
`render json: object, status: status`

:)